### ISSUE TITLE
Edit own posts

### ISSUE THREAD

**Issue by pmuens** [GravityProject/gravity#16]

**Comment by mattvella07** [comment]
@pmuens Are you working on this?  If not, I can work on it.

**Comment by pmuens** [comment]
Yeah I started something but was not quire sure how to solve it (e.g. inline editing vs. new page).

What UX do you prefer?

**Comment by mattvella07** [comment]
That's a good question, I would say inline editing.

**Comment by pmuens** [comment]
Yes, I would prefer that as well. It's also faster than switching between different views and we can use it at other places!

Do you use a package or implement it on your own?

**Comment by mattvella07** [comment]
I haven't looked at it too much, but I was thinking about implementing it using a template to hold a textarea and loading that template when the edit button is clicked.

**Comment by pmuens** [comment]
That's a good idea! :+1:

**Comment by mattvella07** [comment]
Thanks!  There's still a lot of meteor packages I'm not that familiar with.
